keel-core/src/test/kotlin/com/netflix/spinnaker/keel/events/ResourceStatusTests.kt

```diff
@@ -32,25 +32,25 @@ import com.netflix.spinnaker.keel.persistence.ResourceStatus.HAPPY
 import com.netflix.spinnaker.keel.persistence.ResourceStatus.PAUSED
 import com.netflix.spinnaker.keel.persistence.ResourceStatus.RESUMED
 import com.netflix.spinnaker.keel.persistence.ResourceStatus.UNHAPPY
-import com.netflix.spinnaker.keel.persistence.memory.InMemoryPausedRepository
 import com.netflix.spinnaker.keel.services.ResourceStatusService
-import com.netflix.spinnaker.keel.test.combinedInMemoryRepository
 import com.netflix.spinnaker.keel.test.resource
 import com.netflix.spinnaker.time.MutableClock
 import dev.minutest.junit.JUnit5Minutests
 import dev.minutest.rootContext
 import io.mockk.every
-import io.mockk.slot
+import io.mockk.mockk
 import strikt.api.expectThat
 import strikt.assertions.isEqualTo
 
 internal class ResourceStatusTests : JUnit5Minutests {
   object Fixture {
     val clock = MutableClock()
-    val repository = combinedInMemoryRepository(clock)
-    val pausedRepository = InMemoryPausedRepository()
-    val actuationPauser = ActuationPauser(repository.resourceRepository, pausedRepository, repository.publisher, clock)
-    val resourceHistoryService = ResourceStatusService(repository.resourceRepository, actuationPauser)
+    val repository = mockk<ResourceRepository>()
+    val actuationPauser = mockk<ActuationPauser>()
```

Comment:
I thought it made more sense to stub the `ActuationPauser` rather than all its dependencies, since it has its own suite of tests